openedx_events/event_bus/__init__.py

```diff
@@ -157,7 +157,9 @@ def consume_indefinitely(self) -> None:
 #   an instance of EventBusConsumer, calls to the consumer will be ignored with a warning at startup.
 
 
-def make_single_consumer(*, topic: str, group_id: str, signal: OpenEdxPublicSignal, **kwargs) -> EventBusConsumer:
+def make_single_consumer(*, topic: str, group_id: str,
+                         signal: OpenEdxPublicSignal = None,  # pylint: disable=unused-argument
+                         **kwargs) -> EventBusConsumer:
     """
     Construct a consumer for a given topic, group, and signal.
```

Comment:
1. Do you want to drop `signal` from here now?
2. Should we add a sentence stating that the signals will be determined by the message header, or do we not even need to mention signals in the future?

Reply:
1. Going to keep signal here for now since this is part of the public API. I don't know of anyone who uses it but I've broken things before by unexpectedly changing a public API method
2. I think we should just not mention it